src/drt/src/db/tech/frConstraint.h

```diff
@@ -426,7 +426,7 @@ class frMinStepConstraint : public frConstraint
         "Min step length min {} type {} max {} "
         "insideCorner {} outsideCorner {} step {} maxEdges {}",
         minStepLength,
-        minstepType,
+        int(minstepType),
```

Comment:
Is this an spdlog version issue?  It generates no error in our build.

Reply:
I'm on spdlog 1.9.2

from past experience; different compiler versions have different policies on whether or not enums can be implicitly converted to ints, that might be the problem here rather than something in spdlog